### DISCUSSION TITLE
How do I enable support for MyPy and Pylint?

### DISCUSSION THREAD

**Discussion by lucasmsoares96** [helix-editor/helix#9268] [Q&A] [answered]
Hello, everyone. I'm a bit lost on how Helix settings work. How do I enable support for MyPy and Pylint?

**Comment by dusty-phillips** [comment, marked as answer]
I think pylsp supports both, though possibly not out of the box. Here's how to install for Helix purposes: https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#python---pylsp
